### ISSUE TITLE
[JetpackCompose] Add support to share note as Image

### ISSUE THREAD

**Issue by PatilShreyas** [PatilShreyas/NotyKT#119]
In the notes details screen, share as Image option should be provided. 

Question:
How to convert Composables into Bitmap?

**Comment by kasem-sm** [comment]
Hello Shreyas, this can be achieved using PixelCopy but it only supports API 26 and later.

**Comment by ch8n** [comment]
@PatilShreyas Thanks.
I'm not sure but Composable can’t be converted to Bitmap. I will be creating a View and use setContent to the composable, then I write that to bitmap. 

will this be fine?

**Comment by PatilShreyas** [comment]
Yes @ch8n. We can convert composable to view by getting `LocalView` but it captures everything from app bar to bottom of the screen. So it's not good. But yeah, wrapping it in `View` works since it gives us control.

**Comment by ch8n** [comment]
hi @PatilShreyas can you code review I have submitted a PR #269 
also have an article that explains its implementation
https://chetan-garg36.medium.com/jetpack-composable-to-bitmap-image-4ddea2fe3238

**Comment by PatilShreyas** [comment]
Yes @ch8n thanks for raising a PR. Have merged it and will be targetting for release in `v1.2.1`
